etc/profile-m-z/mattermost-desktop.profile

```diff
@@ -0,0 +1,37 @@
+# Firejail profile for mattermost-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include mattermost-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Mattermost
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+
+mkdir ${HOME}/.config/Mattermost
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.config/Mattermost
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+nodvd
+nogroups
+notv
+nou2f
+novideo
+noroot
```

Comment:
Move up (below `nogroups`/`nonewprivs`).

Reply:
Its missing in the latest commit. Forgotten or broken?

Do `seccomp` and `protocol` break?

Reply:
Mattermost is Electron-based app so I removed them as you recommended in Signal MR

Reply:
+1, then the nonewprivs and caps option should also be removed/changed like the command in #2946.